### PR TITLE
Added hermes.radio provider, a project by Rhizomatica Communications.

### PR DESCRIPTION
**Pull request by rafael2k**
Hi all. We are still setting up our email servers, but would like to start testing.

### CODE REVIEW

File: _providers/hermes.radio.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+name: hermes.radio
+status: PREPARATION
```
Comment (r10s) [inline _providers/hermes.radio.md:3]: status `PREPARATION` requires a `before_login_hint` that describes the user what to do to make things working.

if there is nothing to do, just use the status `OK`.

(this has nothing to do with this file or the server is in "preparation", it is about preparations the user has to do on real servers (enable imap for gmx etc.)

**Review by r10s — CHANGES_REQUESTED** [review]
compared to the file in the chatgroup, the server-part is missing - not sure if this is intended.

File: _providers/hermes.radio.md (hunk 1 above)
Comment (rafael2k) [reply _providers/hermes.radio.md:3]: I'll change from preparation to OK.
Server info is not there yet, this is intended.

Comment (rafael2k) [reply _providers/hermes.radio.md]: Updated file with OK in status.

**Review by r10s — APPROVED** [review]

**Comment by r10s** [comment]
yeah, lgtm now, i merge that already in so that the config-defaults will get set. once provider-db is updated in core, it will go to the apps.

**Comment by rafael2k** [comment]
Yay! tks!

**Comment by r10s** [comment]
np, if needed, please add a server section later.

i also just ran the update-script to get it to core: https://github.com/deltachat/deltachat-core-rust/pull/1940

**Pull request merged**